ocaml_emulator/platform.ml

```diff
@@ -11,6 +11,7 @@ let config_enable_dirty_update         = ref false
 let config_enable_misaligned_access    = ref false
 let config_mtval_has_illegal_inst_bits = ref false
 let config_enable_pmp                  = ref false
+let config_enable_rvv                  = ref true
```

Comment:
I doubt this will end well when you try and use floating-point instructions?

Reply:
I'm not familiar with the OCaml part, so could you please offer some suggestions about how to deal with this issue?